### PR TITLE
Added a basic outline for the Landing Page Template

### PR DESCRIPTION
**Pull request by brianzhou13**
haven't tested with a running server, but you can review the outline of how it's setup.

-also added a styles.css on index.html to pair

### CODE REVIEW

File: client/js/landing/landing.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+angular.module('fridgely', [])
```
Comment (chriskwong91) [inline client/js/landing/landing.js]: This should be 'fridgely.landing'

File: client/js/landing/landing.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	.controller('fridgely.landingPageController', function($scope) {
```
Comment (chriskwong91) [inline client/js/landing/landing.js]: The name of the controller shouldn't include 'fridgely.'

**Comment by RSiferd2** [comment]
Starting review

**Comment by chriskwong91** [comment]
reviewing pr

**Pull request merged**